package.json

```diff
@@ -25,49 +25,49 @@
     "graceful-fs": "^4.2.11"
   },
   "dependencies": {
+    "@rollup/plugin-commonjs": "^26.0.1",
+    "@rollup/plugin-node-resolve": "^15.2.3",
+    "@rollup/stream": "^3.0.1",
     "@uswds/uswds": "^3.8.1",
     "cbor-js": "0.1.0",
+    "chart.js": "^4.4.3",
```

Comment:
It looks like `chart.js` is being added back in, but I don't believe we need it anymore, right @heyitsmebev and @jonathanbobel?

```suggestion
```

Reply:
Confirmed in Slack that we don't need this in place - @alexjanousekGSA, would you mind making sure this dependency isn't included?  After that we should be in good shape to merge. 🙂 Thanks!

Reply:
Removed and pushed.